regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -27,19 +29,41 @@ var CommentReviewView = Backbone.View.extend({
 
     this.previewLoading = false;
 
+    this.listenTo(CommentEvents, 'read:proposal', this.handleRead);
+
     this.render();
   },
 
   findElms: function() {
     this.$status = this.$el.find('.status');
   },
 
+  handleRead: function() {
+    var section = this.docId + '-preamble-' + this.docId + '-I';
+    var options = {id: section, section: section, mode: 'read'};
+
+    $('#content-body').removeClass('comment-review-wrapper').addClass('preamble-wrapper');
```

Comment:
Do we need to add the `preamble-wrapper` class here? I don't see references to it in the less or js. Maybe we can drop that class altogether. Also, if we moved the `comment-review-wrapper` class from the `div` to the child `section`, would we have to remove the class at all?